wandb/sdk/launch/_launch.py

```diff
@@ -62,6 +62,7 @@ def resolve_agent_config(  # noqa: C901
     max_jobs: Optional[int],
     queues: Optional[Tuple[str]],
     config: Optional[str],
+    verbosity: Optional[int],
```

Comment:
can this value be None? why is the type annotation Optional? based on the doc string it can only be int.